l1-contracts/test/portals/UniswapPortal.t.sol

```diff
@@ -72,7 +74,7 @@ contract UniswapPortalTest is Test {
   function _createDaiWithdrawMessage(address _recipient, address _caller)
     internal
     view
-    returns (bytes32 entryKey)
+    returns (bytes32 l2ToL1Message)
```

Comment:
Writing `l2ToL1Message` here is a little confusing when it is not a message but the hash. 
The name entry-key came from it being an entry in the hash db, but might also be stupid atm hehe.